routes/index.js

```diff
@@ -1,4 +1,8 @@
 const express = require('express');
 const router = express.Router();
 
+const productRouter = require('./productRouter');
```

Comment:
```suggestion
const express = require('express');

const productRouter = require('./productRouter');

const router = express.Router();
```
필요한 모듈들을 불러온 후, 이를 사용하는 순으로 작성되는게 일반적인 코딩 컨벤션입니다.